qiskit/quantum_info/synthesis/clifford_decompose.py

```diff
@@ -65,699 +52,12 @@ def decompose_clifford(clifford, method=None):
     num_qubits = clifford.num_qubits
 
     if method == "AG":
-        return decompose_clifford_ag(clifford)
+        return synth_clifford_ag(clifford)
 
     if method == "greedy":
-        return decompose_clifford_greedy(clifford)
+        return synth_clifford_greedy(clifford)
 
     if num_qubits <= 3:
-        return decompose_clifford_bm(clifford)
-
-    return decompose_clifford_greedy(clifford)
-
-
-# ---------------------------------------------------------------------
-# Synthesis based on Bravyi & Maslov decomposition
-# ---------------------------------------------------------------------
-
-
-def decompose_clifford_bm(clifford):
```

Comment:
Removing these public functions are API breaking, so could you add deprecation warning?

Reply:
These functions are internal and not part of the API. The only function appearing in the docs is `decompose_clifford`:
https://qiskit.org/documentation/apidoc/quantum_info.html#synthesis

Reply:
Thanks for catching this! I fixed the documentation accordingly.